docs/06-concepts/11-authentication/05-custom-overrides.md

```diff
@@ -42,6 +46,36 @@ List<String> scopes = extractScopes(token);
 Set<Scope> userScopes = scopes.map((scope) => Scope(scope)).toSet();
 ```
 
+### Handling revoked authentication 
+
+When a user's authentication is revoked, the server must be promptly notified to maintain secure access control. The `authenticationRevoked` event plays a crucial role in closing method streams that the user is no longer authorized to access. To trigger this event, invoke the `session.messages.authenticationRevoked` method.
+
+```dart
+await session.messages.authenticationRevoked(
+  userId,
+  RevokedAuthenticationScope(
+    scopes: removedScopes,
+  ),
+);
+```
+
+##### Parameters
+
+- `userId` - The user id for the concerned user. This value should match the `userId` field in the `AuthenticationInfo` object.
+- `RevokedAuthentication` - The extent of the authentication revocation. 
```

Comment:
Suggestion 1: The previous bullet was the parameter _name_, whereas this is the parameter _type_. I would suggest to change this to the parameter name as well (`message` I think?).

Suggestion 2: Be more strictly technical (more like code doc comments):
```suggestion
- `message` - The revoked authentication event message. See below for the different types of messages.
```

Reply:
Updated the example to reflect this as well.